gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/PegasusPlugin.java

```diff
@@ -561,6 +561,8 @@
       "dataTemplateGenerator", "restTools", "avroSchemaGenerator");
   // Directory in the dataTemplate jar that holds schemas translated from PDL to PDSC.
   private static final String TRANSLATED_SCHEMAS_DIR = "legacyPegasusSchemas";
+  // Enable the use of argFiles for the tasks that support them
+  private static final String ENABLE_ARG_FILE = "enableArgFile";
```

Comment:
This is sort of a general question, but I always wonder if having properties like this could collide with other properties in other plugins. Do you think it would make sense to prefix our properties with `pegasus.`?

Reply:
Good point, prefixed with pegasusPlugin